lib/customers.js

```diff
@@ -332,4 +338,40 @@ function batch () {
     }, customers)))
 }
 
-module.exports = {add, get, batch, getById, update}
+function updatePhotoCard (id, data) {
+  let imageData = _.get('idCardPhotoData', data)
+
+  if (_.isEmpty(imageData)) {
+    return Promise.resolve(data)
+  }
+
+  data = _.omit('idCardPhotoData', data)
```

Comment:
We should have a comment that shows what ``data`` looks like. Also, ``data`` shouldn't be mutated, we should define a new variable for the filtered view of data. More functional.

Reply:
ok